source/common/http/http1/codec_impl.cc

```diff
@@ -376,16 +377,90 @@ void ServerConnectionImpl::onEncodeComplete() {
   }
 }
 
+// handlePath computes the correct
```

Comment:
nit: You don't need to duplicate comments here as you have them in the header file. I would just delete.